ivy_tests/test_ivy/test_functional/test_nn/test_layers.py

```diff
@@ -132,7 +132,63 @@ def test_dropout(
         x=x[0],
         prob=prob,
         scale=scale,
-        dtype=dtype,
+    )
+    ret = helpers.flatten_and_to_np(ret=ret)
+    for u in ret:
+        # cardinality test
+        assert u.shape == x[0].shape
+
+
+# dropout1d
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=0,
+        max_value=50,
+        allow_inf=False,
+        min_num_dims=2,
+        max_num_dims=3,
+        min_dim_size=1,
+        max_dim_size=5,
+    ),
+    prob=helpers.floats(min_value=0, max_value=0.9),
+    training=st.booleans(),
+    data_format=st.sampled_from(["NWC", "NCW"]),
+    num_positional_args=helpers.num_positional_args(fn_name="dropout1d"),
+)
+def test_dropout1d(
+    *,
+    dtype_and_x,
+    prob,
+    training,
+    data_format,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    container,
+    instance_method,
+    fw,
+    device,
+):
+    dtype, x = dtype_and_x
+    ret = helpers.test_function(
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        container_flags=container,
+        instance_method=instance_method,
+        fw=fw,
+        fn_name="dropout1d",
+        test_values=False,
+        x=x[0],
+        prob=prob,
+        training=training,
+        data_format = data_format,
+        ground_truth_backend='tensorflow'
     )
     ret = helpers.flatten_and_to_np(ret=ret)
     for u in ret:
```

Comment:
This should be `for u, v, k in zip(ret, gt_ret, x):`, also we should assert the shape across all 3.